pkg/service/volume_service_test.go

```diff
@@ -154,7 +126,10 @@ func (suite *VolumeServiceTestSuite) getDefaultVolumeOptions() *nimbleos.Volume
 	descriptionField := "This volume was created as part of a unit test"
 	var limitIopsField int64 = 256
 	var limitMbpsField int64 = 1
-	folder, _ := suite.nonTenantFolderService.GetFolderByName("default")
+
+	// TODO: replace <foldername> with your folder's name (owned by both admin and tenant)
```

Comment:
Should we have a folder that is created by the test, with a predictable name?

Reply:
That's what I tried to do initially. The problem I ran into was:
1. Tenant doesn't have access to create a folder
2. Admin can create a folder, but this folder is not accessible to tenant
3. Admin can assign the folder to tenant, but it involves using an api with endpoint /api/cns/v1/users instead of the regular /api/tenant/v1/users and I got confused on how to incorporate that to the code.
So me and @raunakkumar  decided to leave it like this and assume tenant and folder id is given (especially since one of the optimization is to replace all placeholder values with environment variables anyway).

Reply:
I'll defer to Raunak.  It does seem like a concern that the SDK doesn't provide access to this API, though.

Reply:
We expect the Tenanadmin SDK to be used by the tenants. The array CLI already exposes a way to create tenants and folders. For now we haven't decided on exposing the admin APIs of the tenant proxy. We can always add that capability once we discuss it internally

Reply:
Gotcha.  Thanks for the explanation.